crowdfunding/crowdfunding/settings.py

```diff
@@ -20,27 +20,43 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-6f5fiv53l$d=%d_0_8&znvd!6&d3rfy-qowzswx^u)i-p_dsm6'
-
+SECRET_KEY = os.environ.get(
+    
+    'DJANGO_SECRET_KEY',
+    'django-insecure-6f5fiv53l$d=%d_0_8&znvd!6&d3rfy-qowzswx^u)i-p_dsm6'
+)
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
-
-ALLOWED_HOSTS = []
+DEBUG = os.environ.get(
+    'DJANGO_DEBUG', 
+    'False'
+) != 'False'
 
+ALLOWED_HOSTS = ['*']
```

Comment:
`ALLOWED_HOSTS = ['*']` is very insecure, and probably shouldn't be used in production systems